src/measure_theory/measure/vector_measure.lean

```diff
@@ -676,6 +676,35 @@ begin
   rw [restrict_empty, restrict_empty]
 end
 
+lemma le_iff_le_restrict_univ : v ≤[univ] w ↔ v ≤ w :=
+begin
+  split,
+  { intros h s hs,
+    have := h s hs,
+    rwa [restrict_apply _ measurable_set.univ hs, inter_univ,
+         restrict_apply _ measurable_set.univ hs, inter_univ] at this },
+  { intros h s hs,
+    rw [restrict_apply _ measurable_set.univ hs, inter_univ,
+        restrict_apply _ measurable_set.univ hs, inter_univ],
+    exact h s hs }
+end
+
+end
+
+section
+
+variables {M : Type*} [topological_space M] [ordered_add_comm_group M] [topological_add_group M]
+variables (v w : vector_measure α M)
+
+lemma neg_le_neg {i : set α} (hi : measurable_set i) (h : v ≤[i] w) : -w ≤[i] -v :=
```

Comment:
can you also deduce from this one a version `neg_le_neg_iff` and add it a simp attribute? (iff versions are often easier to use when rewriting).